website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -57,6 +57,51 @@ Use the Accordion when the toggle or the content is relatively complex.
 ![Accordion with complex content](/assets/components/accordion/usage-do-complex.png =770x*)
 !!!
 
+## Sizes
+
+The Accordion comes in three sizes: `small`, `medium`, and `large`. We recommend using `medium` for most use cases, but use whichever size best fits your UI’s needs.
+
+![Image here of different sizes - only card variant](/assets/components/accordion/accordion-size-range.png)
+
+!!! Dont
+
+The size is set at the Accordion (group) level, don't mix different sizes of the AccordionItem.
+
+![Image here of DONT change sizes - only card variant](/assets/components/accordion/accordion-size-Dont.png)
+!!!
+
+## Types
+
+The Accordion has two container types: `card` and `flush`, and the `static` property, which applies only to the AccordionItem.
```

Comment:
The way this is structured is somewhat confusing because you say "two container types" then include three values/properties. Since you have a section below I might recommend just removing this part, and maybe since you have detailed breakdowns of each property, consider removing this entire sentence.